packages/cli/commands/preview.js

```diff
@@ -0,0 +1,93 @@
+const fs = require('fs');
+const path = require('path');
+const { i18n } = require('../lib/lang');
+const { logger } = require('@hubspot/cli-lib/logger');
+const { getAccountId } = require('../lib/commonOpts');
+const { EXIT_CODES } = require('../lib/enums/exitCodes');
+const { getCwd } = require('@hubspot/cli-lib/path');
+const { preview } = require('@hubspot/cli-lib/lib/preview');
+const { getUploadableFileList } = require('../lib/upload');
+const { trackCommandUsage } = require('../lib/usageTracking');
+
+const i18nKey = 'cli.commands.preview';
+
+const validateSrcPath = src => {
+  const logInvalidPath = () => {
+    logger.error(
+      i18n(`${i18nKey}.errors.invalidPath`, {
+        path: src,
+      })
+    );
+  };
+  try {
+    const stats = fs.statSync(src);
+    if (!stats.isDirectory()) {
+      logInvalidPath();
+      return false;
+    }
+  } catch (e) {
+    logInvalidPath();
+    return false;
+  }
+  return true;
+};
+
+exports.command = 'preview <src> <dest>';
```

Comment:
yeah, and with cms turning into Content Hub -- I'm starting to wonder even more about bucketing things under the "cms" terminology and whether or not that is going to morph as well. 

Either way, I do agree with the sentiment here that this could probably move to `hs cms preview` to distinguish the command is for cms only

Reply:
Yeah, I would agree that this probably makes sense to bucket under `hs cms preview` to distinguish that it is for cms usage only